teamspeak/content.md

```diff
@@ -0,0 +1,131 @@
+# What is TeamSpeak?
+
+TeamSpeak 3 offers the ideal voice communication for online gaming, education and training, internal business communication, and staying in touch with friends and family. Our primary focus is delivering a solution that is easy to use, with high security standards, excellent voice quality, and low system and bandwidth usage.
+
+> [teamspeak.com](https://teamspeak.com/)
+
+%%LOGO%%
+
+# How to use this image
+
+```console
+$ docker run -p 9987:9987/udp -p 10011:10011 -p 30033:30033 -e TS3SERVER_LICENSE=accept %%IMAGE%%
+```
+
+Then you can connect to `localhost` in your TeamSpeak3 client. Please write down the server query password, and server admin privilege key that were generated. There are needed to administrate the TeamSpeak 3 server.
```

Comment:
"There are needed to administrate ..." was probably intended to be "_These_ are needed to administrate ..."